.github/pull_request_template.md

```diff
@@ -1,29 +1,33 @@
-## Ticket  
+<!--
+Please refer to the DEVELOPER.md file for detailed contribution guidelines:
+https://github.com/ORNL/DataFed/blob/master/docs/dev/DEVELOPER.md
+-->
 
-<!--- Put ticket # if JIRA or name if Gitlab and link -->    
+## Ticket
 
-## Description 
+<!--- Put ticket # if JIRA or name if Gitlab and link -->  
+
+## Description
 
 <!--- Describe your changes in detail -->  
 
-## How Has This Been Tested?  
+## How Has This Been Tested?
 
 <!--- Please describe in detail how you tested your changes. -->  
 
 <!--- Include details of your testing environment, and the tests you ran to -->  
 
 <!--- see how your change affects other areas of the code, etc. -->  
 
-## Artifacts (if appropriate):  
+## Artifacts (if appropriate):
 
 <!--- Include videos and pictures that validate your work -->  
 
-## Tasks
+## Checklist
 
-* [ ] - A description of the PR has been provided, and a diagram included if it is a new feature.
-* [ ] - Formatter has been run
-* [ ] - CHANGELOG comment has been added
-* [ ] - Labels have been assigned to the pr
-* [ ] - A reviwer has been added
-* [ ] - A user has been assigned to work on the pr
-* [ ] - If new feature a unit test has been added
+- [ ] Branch name follows convention (see [DEVELOPER.md](../docs/dev/DEVELOPER.md))
+- [ ] PR title follows format: `[<TEAM>-<issue-number>] <Brief description>`
```

Comment:
```suggestion
- [ ] PR title follows format: `[<issue-number>-<team>] <Brief description>`
```

Reply:
I'm honestly not sure of the benefit of adding a TEAM item in the branch name, it is just extra wording in the title, not that it matters much but, what is was the reasoning?